express/blocks/hero-animation/hero-animation.js

```diff
@@ -104,6 +109,41 @@ function adjustLayout($overlay, $attributions, animations, $parent) {
   }
 }
 
+function transformToVideoLink($cell, $a) {
```

Comment:
Didn't we say we want to get rid of the `$` ?

Reply:
Not sure we reached an agreement 😉